README.md

```diff
@@ -49,7 +44,7 @@ Create a json config file with `branch-config` and/or `wctflags` keys:
 ```
 Then run:
 ```
-./run.sh -- -t paper-button -t paper-behaviors
+tattoo -t paper-button -t paper-behaviors
```

Comment:
right, without `-t` it tests all the repos. I'd actually use that as the example